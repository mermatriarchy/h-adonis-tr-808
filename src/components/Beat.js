import React from 'react';

export default function Beat() {
  return (
    <>
      <div>Here's a beat</div>
    </>
  );
}