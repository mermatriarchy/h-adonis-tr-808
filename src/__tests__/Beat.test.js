import React from 'react';
import renderer from 'react-test-renderer';
import Beat from '../components/Beat';

{/* See note in Instrument.test.js about testing.
 /* The below checks that the beat component is rendered correctly. */}

 const props = {
    key: 0,
    isActive: true,
    position: 0,
    currPosition: 1
  } 
  
  it('correctly renders Beat component', () => {  
      const BeatComponent = renderer.create(<Beat {...props}/>).toJSON();
      expect(BeatComponent).toMatchSnapshot();
  });