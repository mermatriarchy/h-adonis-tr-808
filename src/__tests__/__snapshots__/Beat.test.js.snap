// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly renders Beat component 1`] = `
<div
  className="beat outline selected"
/>
`;
